module/unit.sh

```diff
@@ -456,7 +456,7 @@ function ::unit:test() {
                 module=${module%.sh}
                 if [ ${#g_MODULES[@]} -eq 0 -o ${g_MODULES[${module}]--1} -eq 1 ]; then
```

Comment:
The latter check would reduce to `[ -1 -eq 1 ]` should the first variable fail resolution.